src/commands/synthetics/api.ts

```diff
@@ -161,13 +160,22 @@ const getMobileApplicationPresignedURL = (
 
 const uploadMobileApplication = (request: (args: AxiosRequestConfig) => AxiosPromise<void>) => async (
   fileBuffer: Buffer,
-  presignedUrl: string
+  presignedUrlParams: PresignedUrlResponse['presigned_url_params']
 ) => {
+  const form = new FormData()
+  Object.entries(presignedUrlParams.fields).forEach(([key, value]) => {
+    form.append(key, value)
+  })
+  form.append('file', fileBuffer)
+
   await retryRequest(
     {
-      data: fileBuffer,
+      data: form,
+      headers: {...form.getHeaders(), 'Content-Length': form.getLengthSync()},
+      maxBodyLength: Infinity,
+      maxContentLength: Infinity,
```

Comment:
Is there a limit in the backend? (the backend is a cloud provider's object storage? eg. s3)

Reply:
As discussed IRL, this option is internal to Axios and doesn't impact the request.